src/modules/person/domain/dbiam-personenuebersicht.ts

```diff
@@ -75,27 +75,28 @@ export class DbiamPersonenuebersicht {
             organisationIdsForKontexte,
         ); //use Organisation Aggregate as soon as there is one
 
-        const personenUebersichten: DBiamPersonenzuordnungResponse[] | EntityNotFoundError =
+        const result: [DBiamPersonenzuordnungResponse[], Date?] | EntityNotFoundError =
             this.createZuordnungenForKontexte(
                 personenKontexte,
                 rollenForKontexte,
                 organisationsForKontexte,
                 organisationIDs,
             );
-        if (personenUebersichten instanceof EntityNotFoundError) {
-            return personenUebersichten;
+        if (result instanceof EntityNotFoundError) {
+            return result;
         }
 
-        return new DBiamPersonenuebersichtResponse(person, personenUebersichten);
+        return new DBiamPersonenuebersichtResponse(person, result[0], result[1]);
     }
 
     public createZuordnungenForKontexte(
         kontexte: Personenkontext<true>[],
         rollen: Map<string, Rolle<true>>,
         organisations: Map<string, Organisation<true>>,
         organisationIds?: string[],
-    ): DBiamPersonenzuordnungResponse[] | EntityNotFoundError {
+    ): [DBiamPersonenzuordnungResponse[], Date?] | EntityNotFoundError {
```

Comment:
Check if the date there really needs to be optional

Reply:
Yes when there is no Personenkontexte for a person.